tests/helpers/aggregatedAmountsHelper-text.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * aggregatedAmountsHelper-test.js
+ * Created by Lizzie Salita 4/1/19
+ */
+
+import * as AggregatedAmountsHelper from 'helpers/aggregatedAmountsHelper';
+
+describe('Aggregated Amounts helper functions', () => {
+    describe('determineScenario', () => {
+        it('should return "normal" when obligated amount is less than current and potential', () => {
+            const mockAmounts = {
+                _obligation: 50,
+                _combinedCurrentAwardAmounts: 75,
+                _combinedPotentialAwardAmounts: 100
+            };
+            const mockedScenario = AggregatedAmountsHelper.determineScenario(mockAmounts);
+            expect(mockedScenario).toEqual("normal");
+        });
+
+        it('should return "exceedsCurrent" for overspending', () => {
+            const mockAmounts = {
+                _obligation: 75,
+                _combinedCurrentAwardAmounts: 50,
+                _combinedPotentialAwardAmounts: 100
+            };
+            const mockedScenario = AggregatedAmountsHelper.determineScenario(mockAmounts);
+            expect(mockedScenario).toEqual("exceedsCurrent");
+        });
+
+        it('should return "exceedsPotential" for extreme overspending', () => {
+            const mockAmounts = {
+                _obligation: 100,
+                _combinedCurrentAwardAmounts: 50,
+                _combinedPotentialAwardAmounts: 75
+            };
+            const mockedScenario = AggregatedAmountsHelper.determineScenario(mockAmounts);
+            expect(mockedScenario).toEqual("exceedsPotential");
+        });
+
+        it('should return "insufficientData" for negative obligations', () => {
+            const mockAmounts = {
+                _obligation: -55,
+                _combinedCurrentAwardAmounts: 75,
+                _combinedPotentialAwardAmounts: 100
+            };
+            const mockedScenario = AggregatedAmountsHelper.determineScenario(mockAmounts);
+            expect(mockedScenario).toEqual("insufficientData");
```

Comment:
I think this maybe answers my question elsewhere regarding handling negative obligations and you're gonna iterate on this to handle the negative scenario?

Not as familiar as perhaps I should be with the relevant conversations around the scope of this work.